src/busted/outputHandlers/codewars.lua

```diff
@@ -29,9 +29,9 @@ return function(options)
   end
 
   local function onTestFailure(element, parent, message, debug)
-    print('\n<FAILED::>Test Failed')
     -- remove '/home/codewarrior/lua/fixture.lua:\d+: ' from message
-    print('\n<LOG:ESC:>' .. escape(message:sub(message:find(' ') + 1)))
+    local msg = escape(message:sub(message:find(' ') + 1))
+    print('\n<FAILED::>Test Failed<:LF:>' .. msg)
```

Comment:
Maybe it's better without `Test Failed`?

Reply:
Yup, I like it more with just the assertion message, but I was wondering if there's any convention among all reporters but I did not check.

If you dont have any specific practice across all reporters, i think just the message is good.

Reply:
Now it is:

![image](https://user-images.githubusercontent.com/23709795/218879677-8c5a6dfe-7b35-4c56-992e-fc4fa0c695a2.png)

I think it's nicer this way.